build.sbt

```diff
@@ -24,7 +24,22 @@ lazy val tradingHelloService: Project = (project in file("trading-hello-service"
   .settings(
     name := "trading-hello-service",
     libraryDependencies ++= Libraries.logging ++ Libraries.cats ++ Libraries.grpc ++ Libraries.test ++ Seq(
-      Libraries.helloServiceApi,
+      // Libraries.helloServiceApi, // don't pull reruired project from Nexus
       Libraries.pureConfig
     )
   )
+  .dependsOn(tradingHelloServiceApi) // build required project instead of pulling it from Nexus
```

Comment:
I think the comment is not necessary, `dependsOn` is self-explanatory.

Reply:
done